docs/content/documentation/content/page.md

```diff
@@ -112,6 +112,10 @@ weight = 0
 # A draft page is only loaded if the `--drafts` flag is passed to `zola build`, `zola serve` or `zola check`.
 draft = false
 
+# A hidden page is loaded and created, but won't be added to Paginators,
+# set to true to hide page or false to override a parent section's `hidden = true` 
+hidden = false
```

Comment:
Since `hidden` is a `Option<bool>` its good to default to `false` right? I was thinking of leaving it empty, implying the `None` state.
I assumed that people would get confused and since the `None` implies `false` with the inheritence rules specified above it, it should be fine!

Lemme know